src/modules/change-email/controller.js

```diff
@@ -0,0 +1,68 @@
+const event = require('../../lib/event');
+const idm = require('../../lib/connectors/idm');
+const crm = require('../../lib/connectors/crm/entities');
+const { logger } = require('../../logger');
+const scheduledNotifications = require('../../controllers/notifications');
+const changeEmailHelpers = require('./lib/helpers');
+
+/**
+ * call IDM to check authentication details, return result
+ * @param  {Object}  request
+ * @param  {Object}  h
+ */
+const postStartEmailAddressChange = async (request, h) => {
+  const { userId } = request.defra;
```

Comment:
I don't think `request.defra` will exist in the water service - this is only present in the UI project where the user is authenticated.
I think we would need to pass the user ID in the request.payload in the same way as you have done for the password.